addons/api/addon/generated/models/credential.js

```diff
@@ -54,6 +54,13 @@ export default class GeneratedCredentialModel extends BaseModel {
   })
   password_hmac;
 
+  @attr('string', {
+    for: 'username_password',
+    isNestedAttribute: true,
+    description: 'The password for credential.',
+  })
+  password;
+
   @attr('string', {
     for: 'username_password',
```

Comment:
Is the `type` of a user/pass cred store `username_password`?

Reply:
correct